mobile/lib/contracts/tree/contracts.dart

```diff
@@ -0,0 +1,54 @@
+import 'package:json_annotation/json_annotation.dart';
+import 'package:tree_of_a_kind/features/cqrs/cqrs_action.dart';
+
+part 'contracts.g.dart';
+
+@JsonSerializable()
+class TreeItemDTO {
+  final String id;
+  final String photoUri;
+  final String treeName;
+
+  TreeItemDTO({this.id, this.photoUri, this.treeName});
+
+  factory TreeItemDTO.fromJson(Map<String, dynamic> json) =>
```

Comment:
Seems cumbersome, are u sure there's no other way?

Reply:
I don't know about what exactly your talking about, but there isn't any way to generate contracts